resources/views/tools/database/index.blade.php

```diff
@@ -21,7 +21,7 @@
                     <thead>
                         <tr>
                             <th>{{ __('voyager::database.table_name') }}</th>
-                            <th style="text-align:right">{{ __('voyager::database.table_actions') }}</th>
+                            <th style="text-align:right no-sort">{{ __('voyager::database.table_actions') }}</th>
```

Comment:
The database-tool doesn't use a Datatable, so none of your classes will kick in here

Reply:
oh, you're right. Thanks. What should I do now? Create one more commit that removes the class in this file?

Reply:
Yes, `resources/views/tools/database/index.blade.php` shouldn't be changed at all.